lib/logstash/inputs/lumberjack.rb

```diff
@@ -48,7 +51,7 @@ def register
 
     @logger.info("Starting lumberjack input listener", :address => "#{@host}:#{@port}")
     @lumberjack = Lumberjack::Server.new(:address => @host, :port => @port,
```

Comment:
this check is done by the ruby-lumberjack server implementation:
https://github.com/elastic/ruby-lumberjack/blob/master/lib/lumberjack/server.rb#L31